tox.ini

```diff
@@ -170,6 +170,10 @@ skipsdist = True
 deps = pylint==2.5.2
        pytest==5.3.5
        yoti==2.14.0
+       openapi-core==0.13.2
+       openapi-spec-validator==0.2.8
+       aiohttp==3.6.2
```

Comment:
Why are they needed here?

Reply:
because `pylint` gives errors on http_server/connection.py. And they are not installed from `testenv` because we are overriding `deps` here.

Reply:
You need to configure `.pylintrc` to ignore them and revert